src/components/Question/Question.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Prism from 'prismjs';
+
+import '../../assets/css/prism.css';
+
+export default class Question extends Component {
+  componentDidMount() {
+    Prism.highlightAll();
+  }
+
+  render() {
+    const {
+      answerClickHandler,
+      question,
+    } = this.props;
+
+    const {
+      title,
+      description,
+      codeSnippet,
+      codeSnippetLang,
+      answers,
+    } = question;
+
+    const sendAnswer = (answer) => {
+      if (answerClickHandler) {
+        answerClickHandler(answer);
+      }
+    };
+
+    if (!question || !answers) {
+      return null;
+    }
+
+    return (
+      <div>
+        <h1>{title}</h1>
+        {description ? <p>{description}</p> : null}
```

Comment:
Rather than a ternary, wouldn't it be better to write:
```
{description && (
  <p>{description}</p>
)}
```
?

Reply:
it would be and it's neater so thanks